CriteoPublisherSdk/Sources/CR_BidManager.h

```diff
@@ -75,23 +76,26 @@
  * - Returned synchronously using the cache bidding strategy (prefetch / get from cache)
  *
  * @param adUnit The ad unit to request a bid
+ * @param contextData The context for this bid
  * @param responseHandler The block called once bid has been loaded
  * Note: responseHandler is potentially not invoked on main queue
  */
 - (void)loadCdbBidForAdUnit:(CR_CacheAdUnit *)adUnit
+                    context:(CRContextData *)contextData
```

Comment:
Sorry If I wasn't clear, but I meant `withContext:` for these kind of signatures as well (just add a commit don't bother amending each)

Reply:
Done